testing.json

```diff
@@ -0,0 +1,151 @@
+{ "type": "FeatureCollection",
+
+    "features": [
+      { 
+        "type": "Feature",
+        "geometry": {
+          "type": "Point", 
+          "coordinates": [-75.134729, 39.974192]
+          },
+          "properties": {
+            "name": "ArtWell",
+            "address": "100 W Oxford St",
+            "ZIP": "19122",
```

Comment:
Likewise, conventionally, Javascript variables are lower case by default.  This would more typically be `zip`, even for acronyms.  Often times people use all uppercase variables to refer to something with a constant value, like `PI = 3.14`.  Again, it's just conventional and doesn't have a functional impact on the execution of your program - however, following common standards makes it easier for other people to read your code and collaborate (and often for you to come back to long after you've written it).